README.md

```diff
@@ -47,30 +47,32 @@ s3rver --help
 ## Supported clients
 
 Please see [Fake S3s wiki page](https://github.com/jubos/fake-s3/wiki/Supported-Clients) for a list of supported clients.
-When listening on HTTPS with a self-signed certificate, the AWS SDK in a Node.js environment will need ```httpOptions: { agent: new https.Agent({ rejectUnauthorized: false }) }``` in order to allow interaction.
+When listening on HTTPS with a self-signed certificate, the AWS SDK in a Node.js environment will need `httpOptions: { agent: new https.Agent({ rejectUnauthorized: false }) }` in order to allow interaction.
 
 Please test, if you encounter any problems please do not hesitate to open an issue :)
 
 ## Static Website Hosting
 
-If you specify an *indexDocument* then ```get``` requests will serve the *indexDocument* if it is found, simulating the static website mode of AWS S3. An *errorDocument* can also be set, to serve a custom 404 page.
+If you specify an *indexDocument* then `GET` requests will serve the *indexDocument* if it is found, simulating the static website mode of AWS S3. An *errorDocument* can also be set, to serve a custom 404 page.
 
 ### Hostname Resolution
 
-By default a bucket name needs to be given. So for a bucket called ```mysite.local```, with an indexDocument of ```index.html```. Visiting ```http://localhost:4568/mysite.local/``` in your browser will display the ```index.html``` file uploaded to the bucket.
+By default a bucket name needs to be given. So for a bucket called `mysite.local`, with an indexDocument of `index.html`. Visiting `http://localhost:4568/mysite.local/` in your browser will display the `index.html` file uploaded to the bucket.
 
 However you can also setup a local hostname in your /etc/hosts file pointing at 127.0.0.1
 ```
 localhost 127.0.0.1
 mysite.local 127.0.0.1
 ```
-Now you can access the served content at ```http://mysite.local:4568/```
+Now you can access the served content at `http://mysite.local:4568/`
 
 ## Tests
 
-The tests should be run by one of the active LTS versions. The CI Server runs the tests on version `4.x`, `6.x` and `8.x`.
-I recommend using [NVM](https://github.com/creationix/nvm) to manage your node versions.
-  
+The tests should be run by one of the active LTS versions. The CI Server runs the tests on the latest `6.x` and `8.x` releases.
+
+Be wary that Windows is not fully supported due to the way its filesystem works, so most tests that involve deleting objects will fail.
```

Comment:
Not in this PR. If you try running tests on Windows without the changes here, you'd run into issues because of either a) permission issues trying to create a test data directory at `/tmp` or b) `ENOENT` errors when resetting the data because Windows seems to have trouble recursively deleting all the folders created by the previous test. I partially fixed these by using `os.tmpdir()` and by not having the before hook fail if a bucket already exists.

I have a commit that updates syntax to ES2015 across the project and another that has a pretty major refactor of the file system store back end. Incidentally I think I fixed whatever file lock trouble Windows had before in the process.